pkg/apitype/policy.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2016-2019, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package apitype
+
+// CreatePolicyPackRequest defines the request body for creating a new Policy
+// Pack for an organization. The request contains the metadata related to the
+// Policy Pack.
+type CreatePolicyPackRequest struct {
+	Name        string   `json:"name"`
```

Comment:
Is `Name` here a unique ID? Here, and in `Policy` and `RequiredPolicy` below. (Also, same question about why not use a URN from before.)

Reply:
- Name for a PP is unique at the org level. 
- In a policy, the name is unique for that PP/version.
- RequiredPolicy describes a PP. So the name == the name of the PP.

Will add comments to clarify this all.